.github/workflows/ci.yml

```diff
@@ -173,6 +173,12 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18
+      - name: Install protoc
+        uses: arduino/setup-protoc@v2
+        with:
+          version: '21.12'
+          repo-token: '${{ secrets.GITHUB_TOKEN }}'
+        if: '!(github.event_name == ''pull_request'' && matrix.skip_pr)'
```

Comment:
Will this require additional install steps for contributors? If so https://github.com/denoland/manual/blob/main/references/contributing/building_from_source.md should be updated